File: atv-1/java/Saudacao.java
import java.util.Calendar;

class Saudacao {
    public static void main(String[] args) {

        Calendar now = Calendar.getInstance();

        int horas;
        horas = now.get(Calendar.HOUR_OF_DAY);

        if (horas >= 5 && horas <= 11) {
            System.out.println("Bom dia");
        } else if (horas >= 12 && horas <= 18) {
            System.out.println("Boa tarde"); 
        } else if (horas >= 19 && horas <= 23) {
            System.out.println("Boa noite");
        } else {
            System.out.println("Vai dormir, Raphael");
        }
    }
}

File: atv-1/python/Saudacao.py
import datetime

hora = datetime.datetime.now().hour

if (hora >= 5 and hora <= 11):
    print('Bom dia') 
elif (hora >= 12 and hora <= 18):
    print('Boa tarde') 
elif (hora >= 19 and hora <= 23):
    print('Boa noite') 
else:
    print('Vai dormir') 

